src/ui/layout/section-layout.svelte

```diff
@@ -76,7 +76,6 @@
 
   @container layout-query-container (width >= 768px) {
     .layout-wrapper {
-      min-height: 100vh;
```

Comment:
This is an unrelated problem I noticed in production. It is creating extra space in the desktop view that causes the scrollbars to appear unnecessarily.